### ISSUE TITLE
Accessibility Issue on Front Page

### ISSUE THREAD

**Issue by marisabrantley** [SyedImtiyaz-1/Code-Shop#56]
Alt text is missing on the logo/linked image (per WAVE).

I'd appreciate this issue to be assigned to me, with the Hacktoberfest label.

Many thanks!

<img width="1300" alt="Screen Shot 2023-10-05 at 9 07 25 PM" src="https://github.com/SyedImtiyaz-1/Code-Shop/assets/60168324/09d384d4-1132-4cc2-9ce5-97337d61efbe">

**Comment by SyedImtiyaz-1** [comment]
@marisabrantley Great. Please work on this.
